src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -7,9 +7,9 @@ <h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?<
   </p>
   <p>A {{ term }} can be deleted if it:</p>
   <ul>
-    <li>Has never been published,</li>
-    <li>Was created on/after April 1, 2022, and</li>
-    <li>Is not a checked or checker workflow.</li>
```

Comment:
Is the condition right here? 
Should this be displayed for a tool which itself is not a workflow?

Reply:
Yeah, it's wrong, I didn't realize tools could be checked.  Maybe I'll change it to not display for services or notebooks.  I probably should change the last item to say "Is not a checked workflow/tool or checker workflow."  That's kind of a mouthful, maybe "Is not checked or a checker workflow."?

Reply:
Done.